.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-	"semi": false,
+	"semi": true,

```

Comment:
Why this change?

Reply:
And why do you have this file if Prettier is not installed in the project and not mentioned in the workshop either? (package.json)

Reply:
Hi,
semi: true is equal to ; in the end
semi: false is without ;

Because I have prettier installed in my vscode and it conflicts with eslinc. No problem i will remove the file and uninstall prettier.